tracer/src/Datadog.Trace/IAST/Aspects/DebugAspects.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="DebugAspects.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.AppSec.Waf.ReturnTypes.Managed;
+using Datadog.Trace.Iast.Dataflow;
+using Datadog.Trace.Vendors.Serilog;
+using Datadog.Trace.Vendors.Serilog.Core;
+
```

Comment:
nit: use scoped namespaces in new files.

Reply:
Fixed